Response/ChargeSubscriptionResponse.php

```diff
@@ -25,4 +25,14 @@
 
 class ChargeSubscriptionResponse extends ReservationOfFixedAmountResponse
 {
+    public $CaptureResult;
```

Comment:
```suggestion
    /** @var float */
    public $CaptureResult;
```

Reply:
Looks like this was copied from an unapproved version of API-PHP, probably best to have that reviewed and approved first.l